client/components/UpdateBox.jsx

```diff
@@ -47,30 +47,42 @@ class UpdateBox extends React.Component {
   render () {
     return (
       // div
-      <div className='col-md-3'>
+      <div className='col-md-8'>
+        <td className='impression col-md-8 pull-left'>
+          <div>
+            {this.props.impression}
+          </div>
+        </td>      
+
         {!this.state.modalActive && (
           <div className='btn-group' role="group">
-            {/*  delete button */}
+            
+            {/* update button -- do not remove a tags.
+              They are necessary to maintain working buttons while keeping bootstrap styling */}
+                      {/*  delete button */}
             <a onClick={this.handleDelete.bind(this)}>
-              <button className='remove btn btn-default'>
+              <button class="btn" className='remove btn btn-default'>
                 {/* remove button */}
                 <span className='glyphicon glyphicon-remove'></span>
               </button>
             </a>
-            {/* update button -- do not remove a tags.
-              They are necessary to maintain working buttons while keeping bootstrap styling */}
+          
+            {/*  EDIT button */}
             <a onClick={this.openModal.bind(this)}>
-              <button className='update btn btn-default'>
+              <button class="btn" className='update btn btn-default'>
```

Comment:
className is used in react instead of class. besides this btn class is already part of the class in className='update btn btn-default'